support-frontend/assets/pages/subscriptions-redemption/components/productSummary/productSummary.jsx

```diff
@@ -12,7 +12,7 @@ function ProductSummary() {
       <div css={styles.contentBlock}>
         <div css={styles.imageContainer}>
           <GridImage
-            gridId="subscriptionDailyPackshot"
+            gridId="editionsPackshotShort"
```

Comment:
how can I look at this image in the grid?

Reply:
Lookup the id in theGrid.js (it's 278357625f580358e74db1563777aa0b4570072d)
Then stick https://media.gutools.co.uk/images/ at the front of it and that gives you the url to the image:
https://media.gutools.co.uk/images/278357625f580358e74db1563777aa0b4570072d